src/vcpkg-test/registries.cpp

```diff
@@ -429,19 +429,20 @@ TEST_CASE ("registries report pattern errors", "[registries]")
 
     Json::Reader r{"test"};
     auto maybe_conf = configuration_deserializer.visit(r, test_json);
-    const auto& errors = r.errors();
-    CHECK(!errors.empty());
-    REQUIRE(errors.size() == 3);
-    CHECK(errors[0] == "test: error: $.registries[0].packages[1] (a package pattern): \"\" is not a valid package "
-                       "pattern. Package patterns must "
-                       "use only one wildcard character (*) and it must be the last character in the pattern (see " +
-                           docs::registries_url + " for more information).");
-    CHECK(errors[1] ==
+    const auto& lines = r.messages().lines();
+    CHECK(!lines.empty());
+    REQUIRE(lines.size() == 3);
+    CHECK(lines[0].to_string() ==
+          "test: error: $.registries[0].packages[1] (a package pattern): \"\" is not a valid package "
```

Comment:
Note that this text is unchanged; all the lines are green because clang-format decided to rewrap it.